### PR TITLE
373 Add E2E tests for arrays

### PR DESCRIPTION
**Pull request by dantownsend**
Resolves https://github.com/piccolo-orm/piccolo_admin/issues/373

### CODE REVIEW

**Comment by dantownsend** [comment]
I think the tests are failing because it needs what's in the latest Piccolo main branch, which isn't in the latest release.

**Comment by dantownsend** [comment]
I'm not sure why the Playwright tests are failing ... 🤔

File: e2e/test_array_columns.py [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+    Make sure we can submit a form containing array columns, and it gets
+    saved in the database.
+    """
+    login_page = LoginPage(page=page)
+    login_page.reset()
+    login_page.login()
+
+    test_page = AddRowPage(page=page, tablename=ArrayColumns._meta.tablename)
+    test_page.reset()
+
```
Comment (sinisaos) [inline e2e/test_array_columns.py:19]: It looks like we need to wait for Vue to load the page. It works for me locally. Try to add this 
```python
# Let Vue JS finish loading
page.wait_for_timeout(1000)
```
like in `choices` tests.

File: e2e/test_array_columns.py [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+
+def _test_value(page: Page, field: str, value: str):
+    login_page = LoginPage(page=page)
+    login_page.reset()
+    login_page.login()
+
+    test_page = AddRowPage(page=page, tablename=ArrayColumns._meta.tablename)
+    test_page.reset()
+
```
Comment (sinisaos) [inline e2e/test_array_columns.py:57]: It looks like we need to wait for Vue to load the page. It works for me locally. Try to add this
```python
# Let Vue JS finish loading
page.wait_for_timeout(1000)
```
like in `choices` tests.

**Comment by dantownsend** [comment]
@sinisaos That worked - thanks!

**Comment by sinisaos** [comment]
You're welcome :)

**Pull request merged**